Fix: Updated the Submit Button docs to reference newer capabilities (#3354)
rjsf-team/react-jsonschema-form: docs/api-reference/form-props.md

## children

> You can provide custom buttons to your form via the `Form` component's `children`.
> Otherwise a default `Submit` button will be rendered.

Reviewer: I think this wording is a little more clear, what do you think?
```suggestion
You can provide custom buttons to your form via the `Form` component's `children`. If no children are provided, by default a `Submit` button will be rendered.
```

> For other ways of modifying the render of the `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation

Author: ```suggestion
For other ways of modifying the render of default the `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation.
```

> Otherwise a default submit button will be rendered.
> 
> For other ways of modifying the render of the `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation

Author: ```suggestion
Otherwise a default `Submit` button will be rendered.

For other ways of modifying the render of this default `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation
```

> For other ways of modifying the render of this default `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation.

Reviewer: Another suggestion
```suggestion
For other ways to modify the default `Submit` button, see both the [Submit Button Options](./uiSchema/#submitbuttonoptions) and the [SubmitButton Template](../advanced-customization/custom-templates/#submitbutton) documentation.
```
